pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
```

Comment:
The [`tldr wget`](https://github.com/tldr-pages/tldr/blob/master/pages/common/wget.md) uses the exact same "Download a URL to a file" phrase.